appsettings.json

```diff
@@ -1,20 +1,21 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "DataSource=app.db;Cache=Shared"
+    "DefaultConnection": "DataSource=app.db;Cache=Shared",
+    "WhaleSpottingContextConnection": "Server=(localdb)\\mssqllocaldb;Database=whale-spotting;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
I don't think this is right, we're not using the connection string here so we can delete this - we're getting our connection string from a .env file instead

Reply:
deleted all this, still works :)